src/main/java/propertyFiles/Bundle.properties

```diff
@@ -666,6 +668,7 @@ notification.email.createDataset=Your new dataset named {0} (view at {1} ) was c
 notification.email.wasSubmittedForReview={0} (view at {1}) was submitted for review to be published in {2} (view at {3}). Don''t forget to publish it or send it back to the contributor, {4} ({5})\!
 notification.email.wasReturnedByReviewer={0} (view at {1}) was returned by the curator of {2} (view at {3}).
 notification.email.wasPublished={0} (view at {1}) was published in {2} (view at {3}).
+notification.email.publishFailedPidReg={0} (view at {1}) in {2} (view at {3}) could not be published due to a failure to register, or update the Global Identifer for the dataset or one of the files in it. Contact support if this continues to happen. 
```

Comment:
Typo requires correction... "Global Identifier".

Reply:
Hold the line. Might also have some grammar revisions to suggest...

Reply:
OK!